packages/react-components/react-infobutton/src/components/InfoIcon/useInfoIconStyles.styles.ts

```diff
@@ -1,33 +1,95 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
 import type { InfoIconSlots, InfoIconState } from './InfoIcon.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const infoIconClassNames: SlotClassNames<InfoIconSlots> = {
   root: 'fui-InfoIcon',
-  // TODO: add class names for all slots on InfoIconSlots.
-  // Should be of the form `<slotName>: 'fui-InfoIcon__<slotName>`
+  // this className won't be used, but it's needed to satisfy the type checker
+  tooltip: 'fui-InfoIcon__tooltip',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+  base: {
+    display: 'inline-flex',
+    backgroundColor: tokens.colorTransparentBackground,
+    color: tokens.colorNeutralForeground2,
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
+    ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingHorizontalXS),
+
+    [`& .${iconFilledClassName}`]: {
+      display: 'none',
+    },
+    [`& .${iconRegularClassName}`]: {
+      display: 'inline-flex',
+    },
+
+    ':hover': {
+      backgroundColor: tokens.colorTransparentBackgroundHover,
+      color: tokens.colorNeutralForeground2BrandHover,
+
+      [`& .${iconFilledClassName}`]: {
+        display: 'inline-flex',
+      },
+      [`& .${iconRegularClassName}`]: {
+        display: 'none',
+      },
+    },
   },
```

Comment:
You should create these base styles using `makeResetStyles` for better performance.